### PR TITLE
added logging

### PR DESCRIPTION
**Pull request by dg-shadow**

### CODE REVIEW

File: sr_ethercat_hand_config/launch/sr_rosbag_log.launch [hunk 1]
```diff
@@ -7,10 +7,12 @@
...
-  <arg name="log_bag_prefix" default="sr_hand_log"/>
+  <arg name="log_bag_prefix" default="sr_log"/>
 
   <arg name="log_location" value="$(arg log_directory)/$(arg log_bag_prefix)"/>
 
+  <arg name="clear_old_sessions" default="true"/>
+
   <arg name="log_topics" default='-e ".*(controller|debug_etherCAT_data).*" /joint_states /tf /tf_static /rosout /diagnostics /mechanism_statistics'/>
 
   <group unless="$(arg split_logs_by_size)">
```
Comment (toliver) [inline sr_ethercat_hand_config/launch/sr_rosbag_log.launch:18]: It seems that if both split params are false then we don't record at all.

File: sr_ethercat_hand_config/launch/bag_rotate.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+        sorted_bag_files = sorted(bag_files, key=lambda x: os.path.getctime(os.path.join(path, x)))
+
+        while len(sorted_bag_files) > desired_bag_number:
+            os.remove(os.path.join(path, sorted_bag_files[0]))
+            sorted_bag_files.pop(0)
+
+        time.sleep(1)
+
+if __name__ == '__main__':
```
Comment (toliver) [inline sr_ethercat_hand_config/launch/bag_rotate.py:19]: The remover was already tested isn't it?

**Review by toliver — CHANGES_REQUESTED** [review]

File: sr_ethercat_hand_config/launch/bag_rotate.py (hunk 2 above)
Comment (dg-shadow) [reply sr_ethercat_hand_config/launch/bag_rotate.py:19]: yes, and i tested it again at 251 on monday

File: sr_ethercat_hand_config/launch/sr_rosbag_log.launch (hunk 1 above)
Comment (dg-shadow) [reply sr_ethercat_hand_config/launch/sr_rosbag_log.launch:18]: True, but if we're not splitting, we're consuming the whole hard disk, so.... ;) I get what you mean though. Unfortunately, I don't think there's any way to limit size without splitting, and I'm not sure if it's wise to have an option to just let it keep recording.

**Review by toliver — APPROVED** [review]

**Pull request merged**